docs/content/reference/dynamic-configuration/traefik.containo.us_tlsstores.yaml

```diff
@@ -36,9 +36,20 @@ spec:
           spec:
             description: TLSStoreSpec configures a TLSStore resource.
             properties:
+              certificates:
+                items:
+                  description: Certificate holds a secret name for the TLSOption resource.
```

Comment:
this is probably wrong: it's a list, and it has nothing to do with the TLSOption (rather the TLSStore?).
But it should be fixed directly as the godocs on the struct, right?